README.md

```diff
@@ -91,18 +91,55 @@ npm test
 ```
 ### Misc
 
-#### MercadoLibre test account creation
-A command line utility to locally create, authorize and store a MeLi test user account has been provided.
+#### MercadoLibre Test Accounts CLI tool
+A command line utility to locally create, authorize and store a MeLi test user account is provided.
+
+__How to use__
+1. Setup env variables in `.env` file: `bin/cli-meli-acocunts/.env`
+
+```
+# Port to use for oauth server. Suggested '3001'
+PORT=
+
+# Mongo DB URL where to register/retrieve user Accounts.
+MONGODB_URL=
+
+# MercadoLibre Application keys used to create Test accounts & refresh Account access tokens.
+MELI_CLIENT_ID=
+MELI_CLIENT_SECRET=
+```
 
-1. Ensure `.env.test` file exists similar to `.env` for test env variables.
 2. Ensure the configured Mongo DB instance is running.
-3. Then:
+3. Run it:
+
 ```
-node bin/meli-test-account --user=DEV_ACCOUNT_USERNAME
+$ node bin/cli-meli-accounts/ -h
+Usage: cli-meli-accounts [options]
+
+Interactive CLI for MercadoLibre user Accounts management.
+
+Options:
+
+  -V, --version          output the version number
+  -u, --user <nickname>  Run using specified nickname Account keys for MeLi API requests.
+  -h, --help             output usage information
 ```
 
-Where `DEV_ACCOUNT_USERNAME` is the Dev account that will request the test user.
-If `--user=` (or equivalent `-u=`) flag is not specified, will try to find it in environment variable `DEV_ACCOUNT_USERNAME`.
+_Additionaly, it could be run doing_ `npm run meli:test-account -- -u "TEST USER 123"`
+
+> example:
+>  `node bin/cli-meli-accounts -u "TEST USER 123"`
+> would output:
+>
+> ```
+>| Welcome!
+ | ? What do you want to do? (Use arrow keys)
+ | > Create a new Test account
```

Comment:
So... No, turns out it's more than that. Now, it's essentially a small CRUD CLI app to manage authorized MercadoLibre Accounts. 
The goal is to allow the developer to easily and quickly manage the available accounts... for **dev** (aka **testing**) purposes. The "catch" is to facilitate the MeLi Test Accounts creation - otherwise we'd have to issue a POST request to a Mercadolibre endpoint which only returns a `{nickname, password}` and doesn't allow to query existing accounts, the creation is limited to 10 per app client owner/id, there is no way to manually delete, etc.

Reply:
created #47  for this